README.md

```diff
@@ -1,26 +1,55 @@
-# Offscreen Effect Player module
+# Offscreen Effect Player module (OEP-module)
+
+[Description of the **Offscreen Effect Player** on the Banuba SDK website](https://docs.banuba.com/face-ar-sdk-v1/core/cpp_api_overview#offscreen-effect-player)
 
 ## Description
 
-Common submodule for some projects using **offscreen_effect_player**
-The module implements interaction with the effect player. The module consists of six main interfaces:
+Common submodule for projects integrating Banuba SDK using C++. This module is independent of the Banuba SDK and represents the implementation of interfaces for  [**Offscreen effect player**](./interfaces/offscreen_effect_player.hpp). The module is a wrapper over Banuba SDK interfaces and provides necessary interfaces for image processing including postprocessing (orientation change, format conversion), and management of effects. This module demonstrates one of the approaches and simplifies integration to the application.
```

Comment:
A common submodule for projects based on C++ that integrate Banuba SDK. It is independent of the Banuba SDK and consists of implementations of interfaces for...

The module is a wrapper over Banuba SDK interfaces and provides necessary interfaces for image processing including postprocessing (orientation change, format conversion), and effects management. This module demonstrates one of the approaches and simplifies integration with the application.